include/seqan/blast/blast_statistics.h

```diff
@@ -555,6 +573,35 @@ _selectSet(BlastScoringScheme<TScore> & scheme)
     return false;
 }
 
+template <typename TValue, typename TSpec>
+inline bool
+_selectSet(BlastScoringScheme<Score<TValue, ScoreMatrix<AminoAcid, TSpec>>> & scheme)
+{
+    using TScore = Score<TValue, ScoreMatrix<AminoAcid, TSpec>>;
+    using TKAValues = KarlinAltschulValues<TScore>;
+    return _selectSet<TScore, TKAValues>(scheme);
+}
+
+template <typename TValue>
+inline bool
+_selectSet(BlastScoringScheme<Score<TValue, ScoreMatrix<AminoAcid, ScoreSpecSelectable>>> & scheme)
+{
+    using TScoreOrig = Score<TValue, ScoreMatrix<AminoAcid, ScoreSpecSelectable>>;
+    bool ret = false;
+    impl::score::matrixTagDispatch(impl::score::MatrixTags(),
+                                   getScoreMatrixId(seqanScheme(scheme)),
```

Comment:
If it did exist before, just leave it, but the name is kind of confusing.
Maybe something like ```toSeqanScoringScheme``` would be more telling. 😏